docs/user_guide/explanations/reconstructions/center_of_rotation.rst

```diff
@@ -51,14 +50,19 @@ What is Tilt?
 -------------
 
 **Tilt** describes the gradual change in the Center of Rotation (COR) along the axis perpendicular
-to the rotation plane (typically corresponding to the rows in a projection).Without considering tilt,
+to the rotation plane (typically corresponding to the rows in a projection). Without considering tilt,
 the reconstructed slices may not align correctly, causing distortions or streaking artifacts.
 
 Key Points:
   - Tilt is measured in degrees.
   - A tilt value of zero means the rotation axis is vertical, and the COR is the same for every sinogram.
   - Using the COR of the first sinogram and the tilt value, the COR for each subsequent sinogram is interpolated or extrapolated.
 
+Order:
+  1. Determine the COR for the first sinogram (slice 0). This serves as the reference point for all subsequent slices.
+  2. Apply the Tilt value to calculate CORs for the remaining sinograms.
+  3. Refine COR and Tilt values as needed to improve accuracy.
+
```

Comment:
I think this can be removed, these are not the steps that a user would go through.